connectors/byoei.py

```diff
@@ -536,9 +536,9 @@ def ingestion_stats(self):
                 {
                     "bulk_operations": dict(self._bulker.ops),
                     "indexed_document_count": self._bulker.indexed_document_count,
-                    "indexed_document_volume": round(
-                        self._bulker.indexed_document_volume
-                    ),
+                    # return indexed_document_volume in number of MiB
+                    "indexed_document_volume": self._bulker.indexed_document_volume
+                    // (1024 * 1024),
```

Comment:
This does a "floor" instead of rounding to the nearest MB. 
```
>>> 2048576 / (1024*1024)
1.95367431640625
>>> 2048576 // (1024*1024)
1
>>> round(2048576 / (1024*1024))
2
>>> 
```

WDYT about using rounding instead? So that the end-user experience can say "About X mb" instead of "At least X mb"?